netbox/resource_netbox_prefix.go

```diff
@@ -44,6 +44,10 @@ func resourceNetboxPrefix() *schema.Resource {
 				Type:     schema.TypeInt,
 				Optional: true,
 			},
+			"site_id": &schema.Schema{
+            				Type:     schema.TypeInt,
+            				Optional: true,
+            },
```

Comment:
You have to repeat this change in netbox/resource_netbox_available_prefix.go because that resource shares some functions with the prefix resource.